### ISSUE TITLE
Whole Module Optimization?

### ISSUE THREAD

**Issue by codecaffeine** [bwhiteley/JSONShootout#8]
I noticed the test target runs without whole module optimization enabled (since it runs the test in debug mode). I wonder if it would be a good thing to include? Though not every developer builds with it enabled.

**Comment by bwhiteley** [comment]
The scheme is set to run the tests in release mode. 
Also, the model objects are in a separate framework target. I suspect that is more likely where compiler optimizations would make a difference (and in the various mapper frameworks themselves, of course).

**Comment by codecaffeine** [comment]
Maybe I’m doing something wrong, but this was the configuration after a fresh clone and `carthage bootstrap --no-build`. When I changed the Build Configuration to _Release_, I saw a minor speed improvement (but larger than the standard deviation):

<img width="898" alt="screen shot 2016-10-24 at 4 26 51 pm" src="https://cloud.githubusercontent.com/assets/71474/19662715/ce14e89c-9a06-11e6-8d4d-1c7098c4193e.png">

That said, the tests were still in line with your findings. It didn’t magically make Argo fast or anything.
